<launch>

    <!-- launch collision publisher -->
    <node name="collision" pkg="morpheus_teleop" type="collision_publisher.py" output="screen" clear_params="true">
        <param name="group_name" value="test"/>
    </node>

</launch>